Feature/real time service update

MarekBleschke opened the pull request.

Commit:
real time service update

Commit:
update service information and unittests

Commit:
pep8

Commit:
fix mysql encoding error in test

mkurek commented on src/ralph_scrooge/plugins/subscribers/service_environment.py:
```diff
@@ -0,0 +1,176 @@
+@subscriber(topic='updateService')
+def service_environment(event_data):
+    logger.info(
+        u'Start service environment processing for service with name: '
```
import from future instead of using `u''`:
```
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```

mkurek commented on src/ralph_scrooge/plugins/subscribers/service_environment.py:
```diff
@@ -0,0 +1,176 @@
+    service.symbol = event_data['uid']
+    # TODO(mbleschke) - need to set profit center but we have only name in
+    # hermes event.
+    service.profit_center = default_profit_center
```
don't set `default_profit_center` here - service might have already set `profit_center`.

mkurek commented on src/ralph_scrooge/plugins/subscribers/service_environment.py:
```diff
@@ -0,0 +1,176 @@
+def _update_service(event_data):
+    default_profit_center = ProfitCenter.objects.get(pk=1)  # from fixtures
+
+    service, created = Service.objects.get_or_create(ci_uid=event_data['uid'])
```
why not `update_or_create`?

mkurek commented on src/ralph_scrooge/plugins/subscribers/service_environment.py:
```diff
@@ -0,0 +1,176 @@
+        'name', flat=True)
+    )
+
+    # Delete obsolete environments.
```
do not delete environment if it is obsolete - it should be still assigned to service for historical reasons

Commit:
PR comments fix

- use imports from __future__
- simplify _update_service
- don't set profit_center
- don't delete envs
